db/migrate/20181019185328_create_projects.rb

```diff
@@ -0,0 +1,11 @@
+class CreateProjects < ActiveRecord::Migration[5.2]
+  def change
+    create_table :projects do |t|
+      t.string :name, null: false, default: ''
+      t.string :url, null: false, default: ''
```

Comment:
I think you can validate this on logic level as well in `Project` model like
```
validates :name, :url, presence: true
```

Reply:
I thought about validating in a database level was going to be better... but in a second thought I'll validate in both sides (database and model). Or just model? What do you think @Sanchezdav

Reply:
I think a robust application should be validated in both (database and logic) 👌